### ISSUE TITLE
Set the minimum version requirements correctly

### ISSUE THREAD

**Issue by reprogrammer** [vazexqi/CodingSpectator#86]

**Comment by reprogrammer** [comment]
Stas,

In b7aaa14d6b9693899dd4cc1fa186acc81c31e50c, I noticed the following new minimum version requirements in the MANIFEST.MF file of the codingtracker plugin.

<pre>
org.eclipse.team.ui;bundle-version="3.5.101",
org.eclipse.compare;bundle-version="3.5.100"
</pre>


Please confirm that these minimum versions are indeed required and they don't cause issues similar to issue #77.

**Comment by Wanderer777** [comment]
I think the real problem could arise if I used org.eclipse.compare plugin from Eclipse Helios SR1 rather than Eclipse Helios. But I checked that both these versions of Eclipse use the same version of org.eclipse.compare plugin. 

In 6eb2bc77c290ceed61ac3fb85f2b5dda42afa871 I made some changes just to be consistent with other plugins: I removed the minimum version requirements from codingtracker, also I changed the value of the version property of the hacked org.eclipse.compare to 3.9.0.qualifier. 

We will confirm that org.eclipse.compare does not cause any problems when we prepare the next feature patch.
